de.dlr.sc.virsat.model.extension.funcelectrical.ui/src/de/dlr/sc/virsat/model/extension/funcelectrical/ui/wizards/HTMLExportPage.java

```diff
@@ -180,13 +182,27 @@ private void createExportUI() {
         final int widthHint = 250;
         data.widthHint = widthHint;
         destinationField.setLayoutData(data);
+		destinationField.addModifyListener(new ModifyListener() {
+			@Override
+			public void modifyText(ModifyEvent e) {
+				String selectedDirectoryName = destinationField.getText();
+				if (selectedDirectoryName.equals("")) {
+					destination = false;
+				} else {
+					destination = true;

```

Comment:
Just a question: We don't initilize `destination`, will this event always trigger before we want to access the variable?

Reply:
Primitive variables always have a default initialization. A boolean is initialized with false.